### PR TITLE
Disable the edit and add signer button for completed documents

### LINKED ISSUE #37: [DOC-205] Completed Documents should not be editable (for now)
* Documents with status completed should disable the edit and add recipient button
* Adding another recipient with another signature can be added later, to reduce complexity for now

<sub>From [SyncLinear.com](https://synclinear.com) | [DOC-205](https://linear.app/documenso/issue/DOC-205/completed-documents-should-not-be-editable-for-now)</sub>

### PR DESCRIPTION
**Pull request by dephraiim**
Fixes #37

### CODE REVIEW

File: apps/web/pages/documents/[id]/recipients.tsx [hunk 1]
```diff
@@ -216,7 +227,7 @@ const RecipientsPage: NextPageWithLayout = (props: any) => {
                                 Not Sent
                               </span>
                             ) : (
-                              ""
+                              <></>
```
Comment (Mythie) [inline apps/web/pages/documents/[id]/recipients.tsx:230]: This can just be `null`

Comment (Mythie) [reply apps/web/pages/documents/[id]/recipients.tsx:230]: Same for those below

**Review by Mythie — APPROVED** [review]

**Review by ElTimuro — CHANGES_REQUESTED** [review]
Looks good, just 1 thing: Non availible options should be disabled, not vanished, so the user can learn when sth. is available and when it isn't without being "surprised" something is gone.

**Review by ElTimuro — APPROVED** [review]
Looks good

**Pull request merged**